doc/getting_started.md

```diff
@@ -1,723 +0,0 @@
-# Jörmungandr Wallet and Chain interactions
```

Comment:
is this daedalus specific?

Reply:
not really, it's general documentation about the api at an abstract level and its integration with jormungandr, but it's completely out of date (it was written before even the first fund)... I think at this point it may as well be rewritten from scratch.